### PR TITLE
Add get specifications methods

### PR DESCRIPTION
**Pull request by smooresni**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/nitsm-python/blob/master/CONTRIBUTING.md).

### What does this Pull Request accomplish?

Adds the `get_specifications_value` and `get_specifications_values` methods.

### Why should this Pull Request be merged?

Needed to complete #160 

### What testing has been done?

Created new system test to validate methods.

- [x] I have run the automated tests (required if there are code changes)

### CODE REVIEW

File: src/nitsm/tsmcontext.py [hunk 1]
```diff
@@ -1317,3 +1321,19 @@ def publish_per_site(
...
+    # Specifications
+
+    def get_specifications_value(self, namespaced_symbol: str) -> str:
+        """Returns the value calculated for the namespaced_symbol in the Semiconductor Module
+        context specifications file. Raises an exception when the associated specifications file or
+        symbol cannot be found.
+        """
+        return self._context.GetSpecValue(namespaced_symbol)
+
+    def get_specifications_values(self, namespaced_symbols: "_Sequence[str]") -> "_Tuple[str]":
```
Comment (ni-erikcrank) [inline src/nitsm/tsmcontext.py:1334]:
```suggestion
    def get_specifications_values(self, namespaced_symbols: "_Sequence[str]") -> "_StringTuple":
```

**Comment by ni-erikcrank** [comment]
Please update [STATUS.md](https://github.com/ni/nitsm-python/blob/main/STATUS.md) to include information for GetSpecificationsValues.

**Review by ni-erikcrank — CHANGES_REQUESTED** [review]

File: src/nitsm/tsmcontext.py (hunk 1 above)
Comment (smooresni) [reply src/nitsm/tsmcontext.py]: https://github.com/ni/nitsm-python/pull/161/commits/af8d6bfcc1112c73a82393b5d3110e487c787f71

**Comment by smooresni** [comment]
> Please update [STATUS.md](https://github.com/ni/nitsm-python/blob/main/STATUS.md) to include information for GetSpecificationsValues.

I also took the opportunity to fix some of the formatting in `STATUS.md`

**Review by ni-erikcrank — APPROVED** [review]

**Pull request merged**